FastSurferCNN/utils/common.py

```diff
@@ -224,46 +225,46 @@ def pipeline(
 def iterate(
     pool: Executor, func: Callable[[_Ti], _T], iterable: Iterable[_Ti]
 ) -> Iterator[Tuple[_Ti, _T]]:
-    """Iterate over iterable, yield pairs of elements and func(element).
+    """
+    Iterate over iterable, yield pairs of elements and func(element).
 
     Parameters
     ----------
     pool : Executor
-        [MISSING]
+        [MISSING].
     func : Callable[[_Ti], _T]
-        function to use
+        Function to use.
     iterable : Iterable[_Ti]
-        iterable
+        Iterable.
 
     Yields
     ------
      element : _Ti
-        elements
+        Elements
     _T
-        [MISSING]
-
+        [MISSING].
     """
     for element in iterable:
         yield element, func(element)
 
 
 def removesuffix(string: str, suffix: str) -> str:
```

Comment:
remove this function, replace occurances with string.removesuffix as described above in other file.